js/queue/queue-data-structure.js

```diff
@@ -6,23 +6,36 @@ class Queue {
 
   canEnqueue() {
     // ... your code goes here
+    return this.queueControl.length < this.MAX_SIZE ? true : false;
```

Comment:
Cualquier comparación ya resuelve por si misma en un booleano true o false. 
```suggestion
    return this.queueControl.length < this.MAX_SIZE
```